spec/controllers/api/v1/admin/comments_controller_spec.rb

```diff
@@ -0,0 +1,64 @@
+require 'rails_helper'
+
+RSpec.describe Api::V1::Admin::CommentsController, type: :controller do
+  before do
+    allow(controller).to receive(:authenticate_user!).and_return(true)
+    allow(controller).to receive(:current_user).and_return(user)
+  end
+
+  let(:user) { FactoryGirl.create(:user, role: 'admin') }
+  let(:member) { FactoryGirl.create(:user, role: 'member') }
+
+
+  describe "GET #index" do
+    it "allows authenticated access" do
+      get :index, format: :json
+      expect(response).to be_success
+    end
+
+    it "returns user's comments" do
+      FactoryGirl.create(:comment, user_id: user.id, receiver_id: member.id)
+      get :index, params: {receiver_id: member.id}, format: :json
+      expect(JSON.parse(response.body)).not_to be_empty
+    end
+  end
+
+  describe "POST #create" do
+    it "creates a new comment" do
+      comment = FactoryGirl.build(:comment)
```

Comment:
There is a `feature` in factory girl that we can use `build` & `create` methods without `FactoryGirl`. We just need to configure the gem in our project. Can you check the docs?